app/services/analytics_events.rb

```diff
@@ -2990,5 +2990,87 @@ def email_sent(action:, ses_message_id:, **extra)
       **extra,
     )
   end
+
+  def idv_doc_auth_welcome_visited(**extra)
+    track_event('IdV: doc auth welcome visited', **extra)
+  end
+
+  def idv_doc_auth_welcome_submitted(**extra)
+    track_event('IdV: doc auth welcome submitted', **extra)
+  end
+
+  def idv_doc_auth_agreement_visited(**extra)
+    track_event('IdV: doc auth agreement visited', **extra)
+  end
+
+  def idv_doc_auth_agreement_submitted(**extra)
+    track_event('IdV: doc auth agreement submitted', **extra)
+  end
+
+  def idv_doc_auth_upload_visited(**extra)
+    track_event('IdV: doc auth upload visited', **extra)
+  end
+
+  def idv_doc_auth_upload_submitted(**extra)
+    track_event('IdV: doc auth upload submitted', **extra)
+  end
+
+  def idv_doc_auth_document_capture_visited(**extra)
+    track_event('IdV: doc auth document_capture visited', **extra)
+  end
+
+  def idv_doc_auth_document_capture_submitted(**extra)
+    track_event('IdV: doc auth document_capture submitted', **extra)
+  end
+
+  def idv_doc_auth_verify_document_status_submitted(**extra)
+    track_event('IdV: doc auth verify_document_status submitted', **extra)
+  end
+
+  # @identity.idp.previous_event_name IdV: in person proofing ssn visited
+  def idv_doc_auth_ssn_visited(**extra)
+    track_event('IdV: doc auth ssn visited', **extra)
+  end
```

Comment:
[per my RFC](https://docs.google.com/document/d/1WnZ7VSZBLAJTSIquLRBgwKw3blJTt1P5ECsnSzBHMhI/edit#) this is me collapsing two events that are essentially the same into one.

See the [idv/analytics_spec.rb changes](https://github.com/18F/identity-idp/pull/7191/files#diff-754d3d08f4ca722e83851cb76d33fa25f60241d083e10df517ac6075638d28a3R111) for a sense of what difference this would make to the actual flow